core/setup.py

```diff
@@ -63,6 +63,7 @@
     "types-requests",
     "types-setuptools",
     "pytest-mypy",
+    "flake8",
```

Comment:
It seems strange that the tests from `core/tests/test_check_style.py` that require `flake8` weren't triggered / didn't fail in our CI. I think that we need to update our CI config to make sure that the tests are triggered and, maybe, switch from flake8 to ruff (#3353) while we are at it.